py3status/cli.py

```diff
@@ -63,6 +63,8 @@ def parse_cli():
     # i3status config file default detection
     # respect i3status' file detection order wrt issue #43
     i3status_config_file_candidates = [
+        "{}/py3status/py3status.conf".format(xdg_home_path),
+        "{}/py3status/config".format(xdg_home_path),
```

Comment:
I think having one file `~/.config/py3status/config` is enough, no need to introduce multiple alternatives - this one is similar to `~/.config/i3status/config`, let's go with it.

Reply:
I cannot imagine why someone would not like `~/.config/py3status/config` if that's a perfectly reasonable XDG compliant file name, my point was just that we already have a zoo of config locations and adding two more seems excessive, one is enough in my mind. But I don't have a strong opinion, so I approve the PR. Perhaps worth getting others' opinions before merging.

Reply:
I don't have a strong opinion too. I personally use `~/.config/py3status/bottom.conf`... and if I really need to check everything, I will turn on `top.conf` or `test.conf`.

Also, because it's easier to jump open `v py3status.conf` in shell than `v py3 config`... not working or opening wrong one...  so you do `v config py3st conf`. Random examples.

Other than that, I think we should support `~/.config/py3status/py3status.conf` because this PR would make it xdg friendly... At same time, we say we support `py3status.conf` now too. Idk.